### ISSUE TITLE
Make a sphere groups and launcher color constant in certain maps

### ISSUE THREAD

**Issue by TRRESTGHGYTCFTGBCXSZD** [jakubg1/OpenSMCE#130]
In Puzz Loop 2, Every time you start the game, the sphere groups are same. it uses the list to spawn the spheres, and there aren't any randomization involved to change ball spawning.

For Example:

GGGBBGGGBBGGGBBBBRRBBRRBBRRBBRRBBRRBBGGGBBGGGBBGGGBBGGGBBGGGBBGGG
GRRGBBYYYBRRBBRBBGYYRRYGGRRBGGYRRYGGRRBBYYGGBBRRRBYYBGGGRGRG...

**Comment by jakubg1** [comment]
Good idea, I've had this one on my mind for quite a while. Will look into this at some point.

**Comment by jakubg1** [comment]
Partially implemented in https://github.com/jakubg1/OpenSMCE/commit/98c46dd1db4f89d9bb92c77c5d26653fbc432a45, only in Sphere Chains, and not yet in Color Generators for Shooters.
